src/adapter/src/coord/sequencer/inner.rs

```diff
@@ -3181,6 +3191,30 @@ impl Coordinator {
         Ok(Self::send_immediate_rows(rows))
     }
 
+    /// WARNING, ENTERING SPOOKY ZONE 3.0
```

Comment:
BTW, this is not the only such method. Look for `~_optimizer_pipeline` methods in the same file.